server/src/models/species.es6

```diff
@@ -0,0 +1,58 @@
+'use strict';
+
+const Sequelize = require('sequelize');
+const url = require('url');
+
+const sequelizeOptions = {
+  dialect: url.parse(process.env.DATABASE_URL, true).protocol.split(':')[0]
+};
+
+if (
+  url.parse(process.env.DATABASE_URL, true).hostname !== 'localhost' &&
+  url.parse(process.env.DATABASE_URL, true).hostname !== 'fs-intake-postgres'
+) {
+  sequelizeOptions.dialectOptions = {
+    ssl: true
+  };
+}
+
+const sequelize = new Sequelize(process.env.DATABASE_URL, sequelizeOptions);
+
+module.exports = sequelize.define(
+  'species',
+  {
+    id: {
+      type: Sequelize.INTEGER,
+      primaryKey: true,
+      autoIncrement: true,
+      field: 'id',
+      allowNull: false
+    },
+    name: {
+      type: Sequelize.STRING,
+      field: 'name',
+      allowNull: false
+    },
+    webUrl: {
+      type: Sequelize.STRING,
+      field: 'web_url'
+    },
+    photos: {
```

Comment:
I think we should just have the url and then commit the photos into the repository as a static asset.

Reply:
`photos` is not necessary since we are using file name is same as species name, removing this column as a task in sprint-2

Reply:
@lpendem could you please drop a link to which story this task is being added to?

Reply:
I think it was part of the PR for #4

Reply:
https://github.com/nciinc/fs-intake-module/blob/sprint-2-development/server/dba/migrations/29-alter-species.js